docs/eventing/samples/sequence/sequence-reply-to-event-display/README.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Sequence Wired to event-display"
+weight: 20
+type: "docs"
+---
+
+# Using Sequences in series
+
+## Prerequisites
+
+For this example, we'll assume you have set up an `InMemoryChannel`
+as well as Knative Serving (for our functions). The examples use `newbroker`
+namespace, again, if you want to deploy to another Namespace, you will need
+to modify the examples to reflect this.
+If you want to use different type of `Channel`, you will have to modify the
+`Sequence.Spec.ChannelTemplate` to create the appropriate Channel resources.
+
+## Overview
+
+We are going to create the following logical configuration. We create a CronJobSource,
+feeding events to a `Sequence`, then taking the output of that `Sequence` and
+displaying the resulting output.
+
+![Logical Configuration](./sequence-reply-to-event-display.png)
+
+
+## Setup
+
+### Create the Knative Services
+
+Change `newbroker` below to create the steps in the Namespace where you want resources
+created.
+
+```yaml
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: first
+spec:
+  runLatest:
+    configuration:
+      revisionTemplate:
+        spec:
+          container:
+            image: us.gcr.io/probable-summer-223122/cmd-03315b715ae8f3e08e3a9378df706fbb@sha256:17f0bb4c6ee5b1e5580966aa705a51f1b54adc794356f14c9d441d91a26412a3
+            env:
+            - name: STEP
+              value: "0"
+
+---
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: second
+spec:
+  runLatest:
+    configuration:
+      revisionTemplate:
+        spec:
+          container:
+            image: us.gcr.io/probable-summer-223122/cmd-03315b715ae8f3e08e3a9378df706fbb@sha256:17f0bb4c6ee5b1e5580966aa705a51f1b54adc794356f14c9d441d91a26412a3
+            env:
+            - name: STEP
+              value: "1"
+---
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: third
+spec:
+  runLatest:
+    configuration:
+      revisionTemplate:
+        spec:
+          container:
+            image: us.gcr.io/probable-summer-223122/cmd-03315b715ae8f3e08e3a9378df706fbb@sha256:17f0bb4c6ee5b1e5580966aa705a51f1b54adc794356f14c9d441d91a26412a3
+            env:
+            - name: STEP
+              value: "2"
+---
+```
+
+
+```shell
+kubectl -n newbroker create -f ./steps.yaml
+```
+
+### Create the Sequence
+
+Here, if you are using different type of Channel, you need to change the
```

Comment:
wouldnt that be always, w/ the new API/CRDs, there is no "default" channel, AFAIK

Reply:
Yeah, I mean if they don't want to use InMemoryChannel, say they want to use Kafka, they need to set their environment up differently.